docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,24 @@ Contributing to NeoForge
 
 3) Follow the code style of the class you're working in (braces on newlines & spaces instead of tabs in Forge classes, inline brackets in patches, etc).
 
+## Workflow
+
+0. Have preliminary discussions on Discord (`#neoforge-github`)
+1. Fork the repository
+2. Check out your fork
+3. Make a branch
+4. Run `gradlew :neoforge:setup` from the project root
+5. Import project into your IDE (IntelliJ/Eclipse) or Reload Gradle Project
+6. Make your changes, modify the decompiled Minecraft source-code as needed
```

Comment:
I would mention that it is in `projects/neoforge/src`. And maybe add a note that the sources in the `base` project are there for reference and should not be modified.